src/web/Utils/PasswordHashProvider.cs

```diff
@@ -35,13 +35,5 @@ private static (string Hash, string Salt) Generate(byte[] passwordAsBytes, byte[
             byte[] hashBytes = SHA512.Create().ComputeHash(passwordWithSaltBytes.ToArray());
             return (Convert.ToBase64String(hashBytes), Convert.ToBase64String(saltBytes));
         }
```

Comment:
the number of times I have had to copy/paste this code in my career .... it has come to the end it seems.